src/compiler/transformers/esnext.ts

```diff
@@ -212,6 +214,13 @@ namespace ts {
             return visitEachChild(node, noDestructuringValue ? visitorNoDestructuringValue : visitor, context);
         }
 
+        function visitCatchClause(node: CatchClause): CatchClause {
+            if (!node.variableDeclaration) {
+                return updateCatchClause(node, createVariableDeclaration("_ignoredCatchParameter"), node.block);

```

Comment:
Instead of `createVariableDeclaration`, we have a `createUniqueName` that you can call, which relieves you of having to write the error out. I think @RyanCavanaugh was just joking about reserving a variable name (but I'm not sure 😄 ).